### ISSUE TITLE
Missing type UUID

### ISSUE THREAD

**Issue by iherger** [prisma/photonjs#111]
I have the following (simplified) model for UserSessions:
```prisma
model UserSession {
  id                String    @default(cuid()) @id @unique
  user              User      @relation("UserSessions")
  verificationToken String    @default(uuid()) @unique
  sessionToken      String    @default(uuid()) @unique
}
```
When I **omit** the attributes `@default(uuid())` in the two token fields, I get the following errors, when creating a UserSession:
```bash
Argument verificationToken: Got invalid value 'c70d9a84-431b-43b6-8ef5-474d32d7b620' on photon.createOneUserSession. Provided UUID, expected String.
Argument sessionToken: Got invalid value 'f57315e9-a829-4584-8ee6-56fcf950e252' on photon.createOneUserSession. Provided UUID, expected String.
```
When I **add** the attributes `@default(uuid())`, my graphql-yoga server refuses to start up, and produces the error following error message:
```bash
Missing type UUID, did you forget to import a type to the root query or mean ID?
```
I guess this is a bug. If not, what would be the correct way to do this?

**Comment by pantharshit00** [comment]
I think there isn't an error in photon in this case. If I pass a correct UUID v4 to the create operation it correctly recognises the type.

![image](https://user-images.githubusercontent.com/22195362/61050885-a7e36480-a405-11e9-8ebb-a5c13315d226.png)
![image](https://user-images.githubusercontent.com/22195362/61050900-afa30900-a405-11e9-9ed8-efdd7e419417.png)


I think you are using nexus-prisma which has the bug. Can you please tell me your graphql server setup.

**Comment by iherger** [comment]
The original issue arose with nexus-prisma, and graphql-yoga, that's correct.

However, I am able to reproduce it as follows:

Model:
```graphql
model UserSession {
  id                String    @default(cuid()) @id @unique
  user              User      @relation("UserSessions")
  verificationToken String    @unique
  sessionToken      String    @unique
  confirmed         Boolean
  initiatedAt       DateTime
  confirmedAt       DateTime?
  userAgent         String
}
```
Photon:
```ts
import Photon from '@generated/photon';
import { v4 } from 'uuid';

const photon = new Photon();

async function main() {
  const session = await photon.userSessions.create({
    data: {
      verificationToken: v4(),
      sessionToken: v4(),
      confirmed: false,
      user: { connect: { id: 'cjxynv33200001e9kmxyb99u3' } },
      initiatedAt: new Date(),
      userAgent:
        'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.1 Safari/605.1.15',
    },
  });

main();
```

**Comment by pantharshit00** [comment]
I am unable to reproduce this. I am using photon version `prisma2@2.0.0-preview-2, binary version: 4f33077038a490e4b2c1c260f109309e4a447181`

If you can still reproduce this after upgrading your photon client, please provide me with a GitHub repository with the reproduction

**Comment by iherger** [comment]
This can be closed, is has been fixed by #132
